src/Nethermind/Nethermind.Blockchain/Synchronization/ISyncConfig.cs

```diff
@@ -78,6 +78,9 @@ public interface ISyncConfig : IConfig
         [ConfigItem(Description = "Enables SNAP sync protocol.", DefaultValue = "false")]
         public bool SnapSync { get; set; }
 
+        [ConfigItem(Description = "Number of account range partition to create. Increase snap sync request concurrency. Value must be between 1 to 256 (inclusive).", DefaultValue = "8")]
+        int SnapSyncAccountRangePartitionCount { get; set; }
```

Comment:
@asdacap can we use byte instead of int? That would solve the problem of potential to big config value.

Reply:
Byte can't represent 256 (max is 255). Anyway, I added a range check, so it should be fine. https://github.com/NethermindEth/nethermind/pull/5342/commits/58f87147b395396ee61c948fbecbe38c5ec2af42#diff-dfe4817333e66ff3f38d57428d2a1592ec6c666638ee5c62dc5c03fd29d58437R66.